### DISCUSSION TITLE
Event on itemdetail name and description change

### DISCUSSION THREAD

**Discussion by apoorvarajan** [microsoft/Microsoft-Fabric-workload-development-sample#150] [Q&A] [answered]
Hello,

Is there any event that fires when the name and description of an item changes in workspace ?

![Screenshot 2024-11-14 at 2 51 50 PM](https://github.com/user-attachments/assets/48a58d99-8770-4956-8ed7-8d768b7566da)
![Screenshot 2024-11-14 at 2 52 11 PM](https://github.com/user-attachments/assets/4ba8707e-6013-4882-87a6-d19081ee7997)

**Comment by dhrumil83** [comment]
Any update on this?

**Comment by shaym83** [comment, marked as answer]
There's no such event currently.
